config/backend-config/backend-config.go

```diff
@@ -308,3 +310,17 @@ func getNotOKError(respBody []byte, statusCode int) error {
 	}
 	return fmt.Errorf("backend config request failed with %d%s", statusCode, errMsg)
 }
+
+func (d *DestinationT) GetRouterIdentifier() string {
```

Comment:
Are we adding here so RouterIdentifier in backend-config so later we can get the config from control-plane ?

Minor suggestion, as it is more idiomatic in go to avoid the `Get`
```suggestion
func (d *DestinationT) GetRouterIdentifier() string {
```